governance.md

```diff
@@ -0,0 +1,104 @@
+
+**<span style="text-decoration:underline;">Tasks</span>**
+Board members are volunteers that pledge to (as appropriate to their current positions and commitments elsewhere):
+
+- Meet** [once] **a year to review the development of the project (also online).
+- Be consulted for scientific advice by the admins. 
+- Help settle conflicts or roadblocks that may arise on the project development.  
+- Help review student applications of interns that may volunteer or work on the QuTiP project.
+- Try to provide funding for QuTiP development and maintenance.
+- Become ambassadors for scientific open source in general and QuTiP in particular.
+- Take formal decisions to allocate eventual funds, upon recommendation of the Admin Team. 
+
+**<span style="text-decoration:underline;">Composition</span>**
+Initial board members are invited by the admin team, based on their track-record in supporting the project and future helping it grow. The admins elect new board members, in agreement with existing board members or upon the suggestion of the current board.
+
+Additional prospective board members should send an email with their proposed candidature and two brief statements: one explaining their past involvement with the QuTiP project and one with their commitment to the QuTiP for the future, e.g., how they would help and support the project in their activity. The minimum number of board members is **[three]** and maximum is **[seven]**. Board seats are held **[indefinitely]**. The board can decide, by majority vote, to remove a board member for good reasons, e.g., if they are not engaged with the project, do not reply to communications or attend board meetings.   
+
+**<span style="text-decoration:underline;">Decisions</span>**
+Each board member has a vote in decisions, which are taken by a majority vote. 
+
+**<span style="text-decoration:underline;">Communications</span>**
+The board agrees to meet **[once a year]** (also via online meeting). The board is informed at least **[once a year]** by the Admin Team on QuTiP development and other activities (via email in the qutip-board channel). 
+
+### 4. Non-profit Support
+Since 2018, QuTiP is an affiliated project of [NumFOCUS](https://numfocus.org/), a 501(c)(3) nonprofit organization registered in the USA. The mission of NumFOCUS is to promote open practices in research, data, and scientific computing **[Unitary Fund](http://unitary.fund/)** pledges to support the QuTiP project as a fiscally sponsored project. QuTiP remains an affiliated project of NumFOCUS. [Unitary Fund](http://unitary.fund/) is a 501(c)(3) nonprofit organization registered in the USA working to create a quantum technology ecosystem that benefits the most people. Due to its role in the quantum software community, Unitary Fund pledges to: 
```

Comment:
```suggestion
Since 2018, QuTiP is an affiliated project of [NumFOCUS](https://numfocus.org/), a 501(c)(3) nonprofit organization registered in the USA. The mission of NumFOCUS is to promote open practices in research, data, and scientific computing **[Unitary Fund](http://unitary.fund/)** pledges to support the QuTiP project as an affiliated project. QuTiP remains an affiliated project of NumFOCUS. [Unitary Fund](http://unitary.fund/) is a 501(c)(3) nonprofit organization registered in the USA working to create a quantum technology ecosystem that benefits the most people. Due to its role in the quantum software community, Unitary Fund pledges to: 
```